sdk/sdk-client/src/utils/PositionUtils.ts

```diff
@@ -0,0 +1,55 @@
+import { Percentage, type Position, type TokenAmount } from '@summerfi/sdk-common/common'
+import BigNumber from 'bignumber.js'
+
+export class PositionUtils {
+  static getLTV({
+    collateralTokenAmount,
+    debtTokenAmount,
+    collateralPrice,
+    debtPrice,
+  }: {
+    collateralTokenAmount: TokenAmount
+    debtTokenAmount: TokenAmount
+    collateralPrice: string
+    debtPrice: string
+  }): Percentage {
+    // Determine the Collateral Value:
+    const collValue = new BigNumber(collateralTokenAmount.amount).times(collateralPrice)
```

Comment:
I'd prefer to modify `TokenAmount.multiply` to accept a `Price` instead (for non-USD prices) or a Utils to handle this multiplicaiton. If this is needed urgently, at least indicate the quote currency for the price. i.e.: `collateralPriceInUSD`

Reply:
done